src/api/controllers/db-controller.ts

```diff
@@ -315,8 +314,17 @@ export function parsePox2Event(poxEvent: DbPox2Event | DbPox3Event) {
         },
       };
     }
+    case SyntheticPoxEventName.RevokeDelegateStx: {
+      return {
+        ...baseInfo,
+        data: {
+          amount_ustx: poxEvent.data.amount_ustx.toString(),
+          delegate_to: poxEvent.data.delegate_to,
```

Comment:
I'm curious where this part is in the stacks-core codebase (haven't been able to find, would love to peek at that code)

Reply:
The synthetic pox events are at https://github.com/stacks-network/stacks-core/blob/next/pox-locking/src/events.rs
This `revoke` one is WIP https://github.com/stacks-network/stacks-core/pull/4157